stats/DataAggregation.md

```diff
@@ -0,0 +1,71 @@
+# Data Aggregation API Overview
+The stats library allows users to define views to configure how data are aggregated and broken down.
+The core data types used are:
+* `Aggregation`: defines the aggregation type (e.g. `Distribution`, `Sum`, `Mean`).
+* `View`: defines how individual measurements are broken down by tags (similar to labels in some 
+other systems) and aggregated.
+
+## Aggregation
```

Comment:
Client-side percentiles are also a very common ask when the monitoring system doesn't support aggregable percentile derivation from histogram data (which I assume is the purpose of `Distribution`?). They don't seem to fit in one of these types.

Reply:
I filed an issue for this https://github.com/census-instrumentation/opencensus-specs/issues/55. I will address soon.